cli/gen-package/src/Generate.elm

```diff
@@ -328,8 +320,29 @@ generateTypeBuilderRecordHelper block =
                         )
                         |> Just
 
-        Elm.Docs.AliasBlock alias ->
-            Nothing
+        Elm.Docs.AliasBlock { name, tipe } ->
+            case tipe of
+                Elm.Type.Record fields Nothing ->
+                    let
+                        lambdaArgType =
+                            fields
+                                |> List.map (\( fieldName, _ ) -> ( fieldName, expressionType ))
+                                |> Annotation.record
+
+                        lambdaValue arg =
+                            fields
+                                |> List.map
+                                    (\( fieldName, _ ) ->
+                                        ElmGen.field (Elm.string fieldName) (Elm.get fieldName arg)
+                                    )
+                                |> Elm.list
+                                |> ElmGen.record
```

Comment:
This could be rewritten with a single `List.map` followed by a `List.unzip` iff wanted